building_energy_standards_data/database_files/level_2_lighting_space_types.json

```diff
@@ -7432,5 +7432,7439 @@
         "lighting_space_type_target_illuminance_setpoint": 237,
         "lighting_space_type_target_illuminance_setpoint_unit": "Lux",
         "annotation": "illuminance setpoint values have been determined based on \"2019\" on total horizontal illuminance values provided by IES"
```

Comment:
I think that the annotation is missing some words. It should probably be changed to something like that:
`Illuminance setpoint values have been determined based on the \"2019\" total horizontal illuminance values based on IES`